tests/rpm-ostree/README.md

```diff
@@ -0,0 +1,45 @@
+This playbook tests the basic rpm-ostree commands and options.
+
+Core Functionality
+  - rpm-ostree compose
+  - rpm-ostree cleanup
+  - rpm-ostree db
+  - rpm-ostree deploy
+  - rpm-ostree rebase
+  - rpm-ostree rollback
+  - rpm-ostree status
+  - rpm-ostree upgrade
+  - rpm-ostree reload
+  - rpm-ostree initramfs
+  - rpm-ostree install
+  - rpm-ostree uninstall
+
+To be Covered
+  - rpm-ostree db
+
+Covered In Another Test
+  - rpm-ostree install
+  - rpm-ostree uninstall
```

Comment:
The package layering feature has a separate test suite already in atomic-host-tests.  The install/uninstall below is the package layering as options to deploy, upgrade, and rebase